VCs/ItemsVC.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  ItemsVC.swift
+//  To-Do App
+//
+//  Created by Ananth Desai on 18/02/22.
+//
+
+import UIKit
+
+class ItemsVC: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    
+    var index: Int
+    let returnRootView: UITableView = {
+        let table = UITableView()
+        table.register(UITableViewCell.self, forCellReuseIdentifier: "Cell")
+        return table
+    }()
+    
+    init(_ index: Int) {
+        self.index = index
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return sourceData[index].items.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = returnRootView.dequeueReusableCell(withIdentifier: "Cell", for: indexPath)
+        cell.textLabel?.text = sourceData[index].items[indexPath.row]
+        cell.backgroundColor = .white
+        return cell
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        
+        view.addSubview(returnRootView)
+        returnRootView.dataSource = self
+        returnRootView.delegate = self
+        print(sourceData[index].items)
```

Comment:
Print statements shouldn't exist in code. 

Also you have not set any constraints to your subviews. This will break layout for sure when screen has multiple views. So please add it in both screens